test/functorch/test_control_flow.py

```diff
@@ -50,6 +52,57 @@ def _fake_map(f, x, *args):
         zs.append(f(xp, *args))
     return _stack_pytree(zs)
 
+def _fake_while_loop(cond_fn, body_fn, operands):
+    while cond_fn(*operands):
+        operands = body_fn(*operands)
+    return operands
+
+def _while_loop_tests():
+    def simple(x):
+        def cond_fn(x):
+            return x.sum() < 10
+
+        def body_fn(x):
+            return [x + 1]
```

Comment:
yes, the return needs to be a list/tuple. This is because we have a `*`operands in the while_loop_dense call (This is to keep it consistent with torch.cond and also a bit less verbose in syntax). A single output will be unpacked by `*`. Let me add some tests for it.

Reply:
Can we restrict the output return type to tuple?